examples/run/cuda/seeding_example_cuda.cpp

```diff
@@ -25,16 +25,20 @@
 #include <iomanip>
 #include <iostream>
 
+// Boost
+#include <boost/program_options.hpp>
```

Comment:
I would prefer to put the Boost includes before the standard library ones. As in general I want the includes to go "from specific to generic". As it is in the ATLAS coding standards as well...